alicloud/resource_alicloud_kvstore_backup_policy_test.go

```diff
@@ -0,0 +1,97 @@
+package alicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/r-kvstore"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAlicloudKVStoreBackupPolicy_basic(t *testing.T) {
+	var policy r_kvstore.DescribeBackupPolicyResponse
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+
+		// module name
+		IDRefreshName: "alicloud_kvstore_backup_policy.policy",
+
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccKVStoreBackupPolicy_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckKVStoreBackupPolicyExists(
+						"alicloud_kvstore_backup_policy.policy", &policy),
+					resource.TestCheckResourceAttr("alicloud_kvstore_backup_policy.policy", "backup_time", "10:00Z-11:00Z"),
+				),
+			},
+		},
+	})
+
+}
+
+func testAccCheckKVStoreBackupPolicyExists(n string, d *r_kvstore.DescribeBackupPolicyResponse) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return fmt.Errorf("No DB account ID is set")
+		}
+
+		client := testAccProvider.Meta().(*AliyunClient)
+		conn := client.rkvconn
+
+		request := r_kvstore.CreateDescribeBackupPolicyRequest()
+		request.InstanceId = rs.Primary.ID
+		policy, err := conn.DescribeBackupPolicy(request)
+		if err != nil {
+			return fmt.Errorf("Error Describe DB backup policy: %#v", err)
+		}
+		if policy == nil {
+			return fmt.Errorf("Backup policy is not found in the instance %s.", rs.Primary.ID)
+		}
+
+		*d = *policy
+		return nil
+	}
+}
+
```

Comment:
I think there is missing testAccCheckKVStoreBackupPolicyDestroy

Reply:
added.